src/test/java/bio/terra/common/fixtures/ConnectedOperations.java

```diff
@@ -490,20 +516,38 @@ public void retryAcquireLockIngestFileSuccess(
         TimeUnit.SECONDS.sleep(5); // give the flight time to fail a couple of times
         datasetDaoUtils = new DatasetDaoUtils();
         String[] sharedLocks = datasetDaoUtils.getSharedLocks(datasetDao, UUID.fromString(datasetId));
+        if (retryType.equals(RetryType.lock)) {
+            assertEquals("no shared locks after first call", 0, sharedLocks.length);
```

Comment:
In general, the pattern for asserts is (text, actual, comparison)
As written you will get the assert but the a message will be backwards like:
Expected: sharedLocks.length
Actual: 0

Reply:
Nice catch!

Reply:
I have been getting these backwards, so thanks for looking at it!

I think in this case, at least for the junit assert I'm using, this is the pattern:
assertEquals(@Nullable String message, long expected, long actual)

I ran a quick test: 
assertEquals("testing assert equals", 0, 1);
message was: "testing assert equals expected:<0> but was:<1>"

So, I think for this case, it's correct. But I could be going against convention with using this type of assert or something.

Reply:
I don't think we are picky about what kind of asserts to use.
I usually use `assertThat' which has the parameter ordering i wrongly complained about. Just fyi, In my test you would see `assertThat("testing assert equals", sharedLocks.length, equalTo(0))`

Reply:
Ahh yeah, I think that's the one I normally get backwards.